tests/cc/test_usdt_probes.cc

```diff
@@ -293,3 +308,56 @@ TEST_CASE("test listing all USDT probes in Ruby/MRI", "[usdt]") {
     }
   }
 }
+
+// These tests are expected to fail if there is no Ruby with dtrace probes
+TEST_CASE("test probing running Ruby process in namespaces",
+          "[usdt][!mayfail]") {
```

Comment:
@yonghong-song I just tagged this with !mayfail for now, so the failure will be reported but won't cause the run to fail. This feels more correct than just skipping the assertion, which I'd consider to be a testing antipattern.

Overall, I think these checks should actually be run in Jenkins, otherwise what is the point in having them?